python/sglang/srt/layers/quantization/modelopt_quant.py

```diff
@@ -1106,7 +1104,7 @@ def process_weights_after_loading(self, layer: torch.nn.Module) -> None:
                 layer.w13_weight_scale,
             )
 
-            print("Applied flashinfer weight processing for both w13 and w2")
+            logger.info_once("Applied flashinfer weight processing for both w13 and w2")
```

Comment:
Can we just remove these logs? Looks like debugging prints that got left in

Reply:
Agree with removing or making it more meaningful, like `swapped w13 and w2` or `interleaving the weight tensors for flashinfer-trtllm-moe`